src/providers/details/tabs/Credentials/components/utils/getAuthTypeValidateFields.ts

```diff
@@ -0,0 +1,50 @@
+import type { IoK8sApiCoreV1Secret } from '@kubev2v/types';
+
+export const getAuthTypeValidateFields = (secret: IoK8sApiCoreV1Secret, authType: string) => {
+  switch (authType) {
```

Comment:
when u returning a kind of tuple, order matters, this is a possible failure point to future development

Reply:
consider an object when order isnt matter

Reply:
it's just an array of field names for the validated/required fields list. Order isn't matter.
I changed strings to enums though